pulpcore/cli/ansible/repository.py

```diff
@@ -121,13 +120,6 @@ def repository(ctx: click.Context, pulp_ctx: PulpContext, repo_type: str) -> Non
         expose_value=False,
         cls=GroupOption,
     ),
-    type_option(
-        choices={
-            "collection-version": PulpAnsibleCollectionVersionContext,
-            "role": PulpAnsibleRoleContext,
-        },
-        default="collection-version",
-    ),
     href_option,
 ]
 content_json_callback = create_content_json_callback(schema=CONTENT_LIST_SCHEMA)
```

Comment:
I am a bit lost here. This schema is supposed to handle both collections and roles?

Reply:
Yes, since collection and roles have the same identifiers (namespace, name, version).